src/pages/ContentCreator/ModulesPage/CreatorsModules.tsx

```diff
@@ -0,0 +1,58 @@
+import {CardColumns} from 'react-bootstrap';
+import React from 'react';
+import {ModuleCard} from '../../../components/ModuleCard/ModuleCard';
+import {Module} from '../../../types/Module';
+import {getPublicModules} from '../../../api';
+import {Layout} from '../../Layout/Layout';
+import {HorizontallyCenteredSpinner} from '../../../components/util/HorizonallyCenteredSpinner';
+import {Message} from '../../../util/Message';
+
+interface State {
+  modules: Module[];
+  state: 'loading' | 'error' | 'success';
+}
+
+class Explore extends React.Component<{}, State> {
```

Comment:
You copied the explore page. You need to rename the class. The page does not load all public modules, it only loads modules that you own. A user id should be added to a module to show who owns it. If the user is admin then they should see all the modules regardless of who owns it. Also, where is the link to create a new module?  Please follow the wireframe https://balsamiq.cloud/sxh8214/pxlrz7x/r6F01